crates/cli/cli/src/commands/mod.rs

```diff
@@ -30,6 +31,10 @@ pub enum HermesCommand {
     /// Query information about IBC objects
     #[clap(subcommand)]
     Query(query::QueryCommands),
+
+    /// Clear subcommands
+    #[clap(subcommand)]
+    Clear(clear::ClearCommands),
```

Comment:
Do we expect to have any other command under `clear`?

If not, how about having a `clear-packets` command instead of nesting it below `clear`?

Reply:
Not sure we expect other commands under `clear`, in the current Hermes we have the following:

```
hermes-clear 
Clear objects, such as outstanding packets on a channel

USAGE:
    hermes clear <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    help       Print this message or the help of the given subcommand(s)
    packets    Clear outstanding packets (i.e., packet-recv and packet-ack) on a given channel
                   in both directions. The channel is identified by the chain, port, and channel IDs
                   at one of its ends
```